templates/emails/group.monthlyreport.hbs

```diff
@@ -81,14 +81,17 @@ We hope it's useful. Don't hesitate to send us your feedback by replying to this
 {{/each}}
 </table>
 
-<h2>Newest sponsors on OpenCollective</h2>
+{{#if topBackers}}
+<h2>Latest related backers</h2>
 <ul>
-<li><a href="https://opencollective.com/iDoneThis">iDoneThis</a>: they are committing $500/month to open source collectives</li>
-<li><a href="https://opencollective.com/auth0">Auth0</a>: they are sponsoring open source projects related to authentication</li>
+{{#each topBackers}}
+  <li><a href="{{website}}">{{name}}</a> gave {{{donationsString}}}</li>
+{{/each}}
 </ul>
+{{/if}}
 
 {{#if group.related}}
-  <h2>Newest related collectives</h2>
+  <h2>Latest related collectives</h2>
   <ul>
   {{#each group.related}}
     <li><a href="{{publicUrl}}?utm_source=email&utm_campaign=monthlyreport">{{name}}</a>, on a mission to {{mission}} [<a href="{{publicUrl}}?utm_source=email&utm_campaign=monthlyreport#support">support that mission!</a>]</li>
```

Comment:
s/support that mission/support their mission. Or even shorter, s/support that mission/support them